ui.apps/src/main/content/jcr_root/apps/styla/components/content/contenthub/contenthub.html

```diff
@@ -8,4 +8,19 @@
          data-sly-unwrap>
         ${attrs[item] @ context='html'}
     </div>
+    <script>
+        if ( !( window.location.host.indexOf( 'google' ) > -1 ) )
+        {
+            document.getElementById('stylaContentSEO').parentNode.removeChild(document.getElementById('stylaContentSEO'));

```

Comment:
Do we need this initial fallback still by the way? or do the have the data-no-hydration now anyhow always?

Reply:
Since the no-hydration is not using the `stylaContentSeo` class it's not interfering with each other. I would keep both for now...